.vscode/settings.json

```diff
@@ -7,5 +7,9 @@
         "toc",
         "winbuild"
     ],
-    "editor.formatOnSave": true
+    "editor.formatOnSave": true,
+    "go.buildFlags": [
+        "-tags",
+        "integration"
```

Comment:
I'm not familiar with these settings... will this also impact regular go builds?